src/components/home/modals/Send.vue

```diff
@@ -587,6 +587,14 @@ export default {
                 return void 0
             }
             let isValid = false
+            let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
```

Comment:
The same here. `alias` could be null here. check alias is null or not before do for loop.